cfgov/unprocessed/css/molecules/notification.less

```diff
@@ -0,0 +1,96 @@
+/* ==========================================================================
+   Capital Framework
+   Notification Styling
+   ========================================================================== */
+
+
+/* topdoc
+  name: Notifications
+  family: cf-molecules
+  notes:
+    - "Markup and styling for notifications"
+  patterns:
+    - name: Default Usage
+      markup: |
+        <div class="m-notification m-notification__error">
+          <span class="cf-notification_icon
+                       cf-notification_icon__error
+                       cf-icon
+                       cf-icon-error-round"></span>
+          <div class="m-notification_message">
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/3aba3bedb96103bc945bd8755eae7f2b70f676ad